src/Stratis.Bitcoin.Features.TestFeature/Controllers/TestFeatureController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.IO;
+using System.Reflection;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.TestFeature.Controllers
+{
+
+    /// <summary>
+    /// Controller providing TestFeature operation
+    /// </summary>
+    [Route("api/[controller]")]
+    public class TestFeatureController : Controller
```

Comment:
`TestFeatureController` sounds like it's a feature that tests something. I'm being picky but maybe `DummyFeatureController` or `ExampleFeatureController` is more self-explicit.

Reply:
TestFeature is included as a way to demonstrate how new feature specific dashboards might look in the code, for purposes of this code review. Do you think there should be an ExampleFeature that lives in the stratis project as an example for feature developers? I think it probably doesn't belong there.

Reply:
This comment is just about naming. :-)
I find the `Test` prefix confusing. Maybe a prefix such as `Dummy` or `Example` will be more clear.

Reply:
I think this should just be called DashboardController 

So I think we should have a new DashboardFeature that depends on ApiFeature
And the DashboardFeature will have internally a DashboardController that will serve some basic node info.